resources/icons/16px/root-folder.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M2 1V6H1C0.447715 6 0 6.44772 0 7V15C0 15.5523 0.447715 16 1 16H11C11.5523 16 12 15.5523 12 15V12H15C15.5523 12 16 11.5523 16 11V3C16 2.45 15.55 2 14.9999 2H9.42066L7.71051 0.3C7.53049 0.11 7.28047 0 7.00045 0H3.00009C2.45004 0 2 0.45 2 1ZM8.59572 4L6.58378 2H4V6H11C11.5523 6 12 6.44772 12 7V10H14V4H8.59572ZM6.99989 8L3.00011 14L4.99998 14L8.99989 8H6.99989Z" fill="black"/>
+</svg>
```

Comment:
We can optimize these SVGs a bit by removing unnecessary attributes and modifying the path so that we no longer need `fill-rule="evenodd"`.

Result:
```svg
<svg viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
<path d="M2 1V6H1C0.447715 6 0 6.44772 0 7V15C0 15.5523 0.447715 16 1 16H11C11.5523 16 12 15.5523 12 15V12H15C15.5523 12 16 11.5523 16 11V3C16 2.45 15.55 2 14.9999 2H9.42066L7.71051 0.3C7.53049 0.11 7.28047 0 7.00045 0H3.00009C2.45004 0 2 0.45 2 1ZM8.59572 4H14V10H12V7C12 6.44772 11.5523 6 11 6H4V2H6.58378L8.59572 4ZM6.99989 8H8.99989L4.99998 14H3.00011L6.99989 8Z"/>
</svg>

```